src/QuickMarcEditor/QuickMarcEditor.test.js

```diff
@@ -49,7 +50,7 @@ jest.mock('./QuickMarcRecordInfo', () => {
 });
 
 const onCloseMock = jest.fn();
-const onSubmitMock = jest.fn();
+const onSubmitMock = jest.fn(() => Promise.resolve({ relatedMarcVersion: 1 }));
```

Comment:
Isn't this parameter just called `version`? It's mocking return value from `QuickMarcEditWrapper` on line 128, right?

Reply:
Yes, its an updated version to update `relatedRecordVersion` in URL. Thanks for noticing that